Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM librecores/librecores-ci
```

Comment:
Should be replaced by LCCI Tools Base later. Just wanted to have an image with ready python bits